kotlin-web-spring-boot-3/src/main/kotlin/com/vndevteam/kotlinwebspringboot3/application/logging/Slf4jMDCFilterConfiguration.kt

```diff
@@ -0,0 +1,30 @@
+package com.vndevteam.kotlinwebspringboot3.application.logging
+
+import org.springframework.boot.web.servlet.FilterRegistrationBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+
+@Configuration
+class Slf4jMDCFilterConfiguration {
```

Comment:
```suggestion
class Slf4jMDCFilterConfig {
```
@haiks-2382 Mình đang dùng hậu tố `Config` cho ngắn gọn